src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -615,7 +615,7 @@ private void LookupForMatchingGitHubRepository(BuildData mergedBuild)
         /// <returns>An AssetData with data about the merge manifest</returns>
         private AssetData GetManifestAsAsset(IImmutableList<AssetData> assets, string location, string manifestFileName)
         {
-            string repoName = GetAzDevRepositoryName();
+            string repoName = GetAzDevRepositoryName().Replace('/','-'); 
```

Comment:
Can you check that you don't need to trim a trailing /?

Reply:
I did see Nuget/Nuget.Client, so I thought this should be good.. I will add it anyway :)

Reply:
It was just based on my spot check of the previous MergedManifest from your test build.

Reply:
oh okay :) do you think I should test this again in Nuget.Client before we merge this?